pkgs/applications/networking/browsers/ladybird/default.nix

```diff
@@ -40,15 +53,27 @@ in gcc11Stdenv.mkDerivation {
   buildInputs = [
     libxcrypt
     qtbase
+  ] ++ lib.optionals stdenv.isDarwin [
+    AGL
+    AppKit
   ];
 
   cmakeFlags = [
-    "-DSERENITY_SOURCE_DIR=${serenity}"
     # Disable network operations
     "-DENABLE_TIME_ZONE_DATABASE_DOWNLOAD=false"
     "-DENABLE_UNICODE_DATABASE_DOWNLOAD=false"
   ];
 
+  preConfigure = ''
+    export cmakeFlags="$cmakeFlags -DSERENITY_SOURCE_DIR=$PWD/serenity"
+  '';
```

Comment:
Doesn't `"-DSERENITY_SOURCE_DIR=$(PWD)/serenity"` in `cmakeFlags` have the same effect?

Reply:
Nope, `$(PWD)/serenity` won't be evaluated in this case, causing a CMake error.

Reply:
Is it not possible to make the path relative?

Reply:
@jiegec Neither, `${placeholder "src"}` points to the unpatched (and unpatchable) source.

Reply:
@Atemu In fact `serenity` works, just with a warning that it's not absolute. Should I switch to that then?